From b975de65196de8d9c592893e117e356a5526c8e6 Mon Sep 17 00:00:00 2001
From: Patrice Chotard <patrice.chotard@foss.st.com>
Date: Thu, 16 Nov 2023 16:45:45 +0100
Subject: [PATCH 0623/1141] arm64: dts: st: enable secure arm-wdt watchdog on
 stm32mp257f-dk

Enable the watchdog and define the default timeout to 32 seconds.

Signed-off-by: Olivier BIDEAU <olivier.bideau@st.com>
Signed-off-by: Patrice Chotard <patrice.chotard@foss.st.com>
Change-Id: I0973cc1f7eb776e1b9a70523cdf41ab8a0f43d4c
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/340492
Reviewed-by: Amelie DELAUNAY <amelie.delaunay@foss.st.com>
Domain-Review: Amelie DELAUNAY <amelie.delaunay@foss.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Antonio Maria BORNEO <antonio.borneo@foss.st.com>
---
 arch/arm64/boot/dts/st/stm32mp257f-dk.dts | 5 +++++
 1 file changed, 5 insertions(+)

--- a/arch/arm64/boot/dts/st/stm32mp257f-dk.dts
+++ b/arch/arm64/boot/dts/st/stm32mp257f-dk.dts
@@ -30,6 +30,11 @@
 	};
 };
 
+&arm_wdt {
+	timeout-sec = <32>;
+	status = "okay";
+};
+
 &scmi_regu {
 	scmi_vddio1: voltd-vddio1 {
 		regulator-min-microvolt = <1800000>;
